character-set-and-collation.md

```diff
@@ -6,7 +6,47 @@ aliases: ['/docs/dev/character-set-and-collation/','/docs/dev/reference/sql/char
 
 # Character Set and Collation
 
-A character set is a set of symbols and encodings. A collation is a set of rules for comparing characters in a character set.
+This document introduces the character sets and collations supported by TiDB.
+
+## Concepts
+
+A character set is a set of symbols and encodings. The default character set in TiDB is utf8mb4, which matches the default in MySQL 8.0 and above.
+
+A collation is a set of rules for comparing characters in a character set, and the sorting order of characters. For example in a binary collation `A` and `a` do not compare as equal:
+
+{{< copyable "sql" >}}
+
+```sql
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+SELECT 'A' = 'a';
+SET NAMES utf8mb4 COLLATE utf8mb4_general_ci;
+SELECT 'A' = 'a';
+```
+
+```sql
+mysql> SELECT 'A' = 'a';
```

Comment:
Please remove the leading `mysql>`

Reply:
Done

Reply:
Why removed it? We use this leading `mysql>` a lot in our documents.

Reply:
According to @TomShawn, if we use this leading `mysql>`. Then if users copy this SQL, they need to trim the leading `mysql>` again.

Reply:
[lines 17-24](https://github.com/pingcap/docs/pull/3402/files#diff-c8c1ce90ea28120a4eb9bd98c70a9213R17-R24) is the SQL code for copying and the leading `mysql>` needs to be removed. 

lines 26-45 are like examples of the execution result, so no need to delete the leading `mysql>`. @wjhuang2016